xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd

```diff
@@ -662,7 +662,11 @@ Rail transport, Roads and Road transport
 			<xsd:element ref="TypeOfFareProductRef" minOccurs="0"/>
 			<xsd:element ref="TypeOfUsageParameterRef" minOccurs="0"/>
 			<xsd:element ref="TypeOfConcessionRef" minOccurs="0"/>
-			<xsd:element ref="TypeOfProofRef" minOccurs="0"/>
+			<xsd:element ref="TypeOfProofRef" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>DEPRECATED. Use typesOfProofAccededRef. -V2.0</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>DEPRECATED. Use typesOfProofAcceptedRef. -V2.0</xsd:documentation>
```

Reply:
Has something gone wrong? I still see the same.

Reply:
I had do do somehing idfferently. What are you unhappy about?